var/spack/repos/builtin/packages/vasp/package.py

```diff
@@ -46,6 +46,8 @@ class Vasp(MakefilePackage):
         "https://github.com/henniggroup/VASPsol",
     )
 
+    variant("shmem", default=True, description="Enable use_shmem build flag")
```

Comment:
Do you know if this option applies to all (at least listed) versions or from a certain version on?

Reply:
I'm not sure, I've only worked with v6+ (and don't have access to v5) but adding this variant does not add any "risk".

Given the current package applies that flag to all versions, if it does not apply then some listed versions would not work as-is today. Given the way the current package always applies that flag, I assumed it was compatible with all listed versions and this just lets us turn it off